tests/rptest/tests/log_level_test.py

```diff
@@ -23,6 +23,22 @@ def __init__(self, *args, **kwargs):
         # it will start.
         super().__init__(*args, log_level=self.initial_log_level, **kwargs)
 
+    @cluster(num_nodes=3, log_allow_list=["Set log level for {assert}"])
+    def test_get_loggers(self):
+        admin = Admin(self.redpanda)
+        node = self.redpanda.nodes[0]
+        loggers = admin.get_loggers(node)
+        assert len(loggers) >= 42, f"Only {len(loggers)} loggers: {loggers}"
```

Comment:
Where did 42 come from? Is it the number of loggers redpanda generally registers?

Reply:
Yeah, that's the current number of loggers. Changed this per Noah's feedback.